GitUI/CommandsDialogs/BrowseDialog/DashboardControl/Dashboard.cs

```diff
@@ -26,321 +26,218 @@ public partial class Dashboard : GitModuleControl
         private readonly TranslationString issues = new TranslationString("Issues");
         private readonly TranslationString openRepository = new TranslationString("Open repository");
         private readonly TranslationString translate = new TranslationString("Translate");
-        private readonly TranslationString directoryIsNotAValidRepositoryCaption = new TranslationString("Open");
-        private readonly TranslationString directoryIsNotAValidRepository = new TranslationString("The selected item is not a valid git repository.\n\nDo you want to abort and remove it from the recent repositories list?");
-        private readonly TranslationString directoryIsNotAValidRepositoryOpenIt = new TranslationString("The selected item is not a valid git repository.\n\nDo you want to open it?");
-        private readonly TranslationString _showCurrentBranch = new TranslationString("Show current branch");
-        private bool initialized;
+        private readonly TranslationString showCurrentBranch = new TranslationString("Show current branch");
+        // ReSharper restore InconsistentNaming
+        private DashboardTheme _selectedTheme;
+
+        public event EventHandler<GitModuleEventArgs> GitModuleChanged;
+
 
         public Dashboard()
         {
             InitializeComponent();
             Translate();
 
-            RecentRepositories.DashboardItemClick += dashboardItem_Click;
-            RecentRepositories.RepositoryRemoved += RecentRepositories_RepositoryRemoved;
-            RecentRepositories.DisableContextMenu();
-            RecentRepositories.DashboardCategoryChanged += dashboardCategory_DashboardCategoryChanged;
-            //Repositories.RepositoryCategories.ListChanged += new ListChangedEventHandler(RepositoryCategories_ListChanged);
-
-            Bitmap image = Lemmings.GetPictureBoxImage(DateTime.Now);
-            if (image != null)
-            {
-                pictureBox1.Image = image;
-            }
-
-            // Do this at runtime, because it is difficult to keep consistent at design time.
-            pictureBox1.BringToFront();
-            pictureBox1.Location = new Point(this.Width - pictureBox1.Image.Width - 10, this.Height - pictureBox1.Image.Height - 10);
+            recentRepositoriesList1.GitModuleChanged += OnModuleChanged;
 
-            Load += Dashboard_Load;
+            ApplyTheme();
```

Comment:
Yes, I didn't run unit tests I must admit... :man_facepalming: 
I'll look into it tomorrow